inbox/types.go

```diff
@@ -221,14 +239,14 @@ type Object interface {
 }
 
 type ObjectIdentifier struct {
-	ObjectName          string
-	ObjectUniqueID      string
-	ObjectType          ObjectTypeIdentifier
-	IsGroupType         bool
-	ParentObject        *ObjectIdentifier
-	AdditionalClaims    *jwt.MapClaims
-	BoundedUserIdentity *UserIdentity
-	AddressAlias        string
+	ObjectName          string               `json:"ObjectName"`
+	ObjectUniqueID      string               `json:"ObjectUniqueID"`
+	ObjectType          ObjectTypeIdentifier `json:"ObjectType"`
+	IsGroupType         bool                 `json:"IsGroupType"`
+	ParentObject        *ObjectIdentifier    `json:"ParentObject,omitempty"`
+	AdditionalClaims    *jwt.MapClaims       `json:"AdditionalClaims,omitempty"`
+	BoundedUserIdentity *UserIdentity        `json:"BoundedUserIdentity,omitempty"`
+	AddressAlias        string               `json:"AddressAlias,omitempty"`
```

Comment:
Remove the json tags

Reply:
These tags are necessary for generating the deepcopy functions for the "SubscriptionResponse" type in Inbox-Agent. Without the tags, it would break the deepcopy functions whenever we run "make gen fmt".